core/src/components/item/item.md.scss

```diff
@@ -348,14 +235,6 @@
 // }
 
 
-// Material Design Stacked & Floating Inputs
-// --------------------------------------------------
-
-:host(.item-label-floating),
-:host(.item-label-stacked) {
-  --min-height: 55px;
-}
```

Comment:
Why were these removed if we're keeping the `ion-label` position styles?

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/29102/commits/02e5958b825b3a07174b178fd29b80b34feec0be